shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/task/MySQLIncrementTask.java

```diff
@@ -61,14 +62,16 @@ public void run() {
     private Object insertOrder() {
         ThreadLocalRandom random = ThreadLocalRandom.current();
         String status = random.nextInt() % 2 == 0 ? null : "NOT-NULL";
-        Object[] orderInsertDate = new Object[]{keyGenerateAlgorithm.generateKey(), random.nextInt(0, 6), random.nextInt(0, 6), random.nextInt(1, 99), status};
+        Object[] orderInsertDate = new Object[]{keyGenerateAlgorithm.generateKey(), ScalingCaseHelper.SNOWFLAKE_KEY_GENERATE_ALGORITHM.generateKey(), random.nextInt(0, 6), 
+                random.nextInt(1, 99), status};
```

Comment:
1, There're 2 key generate algorithm, could we just use one? Just like general IT.

2, It's better to encapsulate `ScalingCaseHelper.SNOWFLAKE_KEY_GENERATE_ALGORITHM.generateKey()` if it's necessary.

Reply:
Now key generate algorithm used SnowflakeKeyGenerateAlgorithm and UUIDKeyGenerateAlgorithm, may not be able to use only one now.